index.html

```diff
@@ -1,33 +1,209 @@
-            <li><a href="#"><img src="images/icons/Linkedin icon.svg" alt="twitter"></a></li>
-            <li><a href="#"><img src="images/icons/Media.svg" alt="twitter"></a></li>
-            <li><a href="#"><img src="images/icons/Github.svg" alt="twitter"></a></li>
-            <li><a href="#"><img src="images/icons/Sideshow.svg" alt="twitter"></a></li>
+  <header class="header">
+    <a href="#" class="logo">My Logo</a>
+    <nav>
+      <i class="material-symbols-outlined menu_icon">
+        menu
+      </i>
+    </nav>
+  </header>
+  <section class="container">
+    <h1>I’m Abdurahim Glad to see you!</h1>
+    <p class="description">I’m a software developer! I can help you build a product , feature or website Look
+      through some of my work and experience! If you like what you see and have a project you need coded, don’t
+      hestiate to contact me.</p>
+    <h2 class="social_media">LET’S CONNECT</h2>
+    <ul class="social_media_icons">
+      <li><a href="#"><img src="images/icons/Twitter.svg" alt="twitter"></a></li>
+      <li><a href="#"><img src="images/icons/Linkedin icon.svg" alt="linkedin"></a></li>
+      <li><a href="#"><img src="images/icons/Media.svg" alt="media"></a></li>
+      <li><a href="#"><img src="images/icons/Github.svg" alt="github"></a></li>
+      <li><a href="#"><img src="images/icons/Sideshow.svg" alt="slideshow"></a></li>
+    </ul>
+  </section>
+  <section class="works_container">
+    <div class="card">
+      <img src="images/Snapshoot Portfolio.png" alt="">
+      <div class="work_desc">
+        <h2>Tonic</h2>
+        <div class="heading_3">
+          <h3 class="main_h3">CANOPY</h3>
+          <span></span>
+          <h3>Back End Dev</h3>
+          <span></span>
+          <h3>2015</h3>
+        </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul>
+          <li>html</li>
+          <li>css</li>
+          <li>javaScript</li>
         </ul>
-    </section>
+        <div class="action">
+          <button class="btn">
+            See Project
+          </button>
+        </div>
+      </div>
+    </div>
+    <div class="card">
+      <img src="images/Snapshoot Portfolio 2.png" alt="">
+      <div class="work_desc">
+        <h2>Multi-Post <br> Stories</h2>
+        <div class="heading_3">
+          <h3 class="main_h3">CANOPY</h3>
+          <span></span>
+          <h3>Back End Dev</h3>
+          <span></span>
+          <h3>2015</h3>
+        </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul>
+          <li>html</li>
+          <li>css</li>
+          <li>javaScript</li>
+        </ul>
+        <div class="action">
+          <button class="btn">
+            See Project
+          </button>
+        </div>
+      </div>
+    </div>
+    <div class="card">
+      <img src="images/Snapshoot Portfolio 3.png" alt="">
```

Comment:
[OPTIONAL] I suggest you Kindly add a value to your `alt` attribute as this will aid accessibility and screen readers will appreciate it😉